Simplenote/src/main/java/com/automattic/simplenote/PreferencesFragment.java

```diff
@@ -197,6 +216,83 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         updateAnalyticsSwitchState();
     }
 
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent resultData) {
+        if (requestCode != REQUEST_EXPORT_DATA || resultCode != Activity.RESULT_OK || resultData == null) {
+            return;
+        }
+
+        if (resultData.getData() != null) {
+            Simplenote currentApp = (Simplenote) requireActivity().getApplication();
+            Bucket<Note> noteBucket = currentApp.getNotesBucket();
+            JSONObject account = new JSONObject();
+            Bucket.ObjectCursor<Note> cursor = noteBucket.allObjects();
+
+            try {
+                JSONArray activeNotes = new JSONArray();
+                JSONArray trashedNotes = new JSONArray();
+                Comparator<String> comparator = new Comparator<String>() {
+                    @Override
+                    public int compare(String text1, String text2) {
+                        return text1.compareToIgnoreCase(text2);
```

Comment:
for whatever value it has we're not sorting by a locale-sensitive sort here. given the complexity of doing locale-aware sorts _and_ locale-aware lower-casing and given that it only controls the tag list in the output I'm sure it's fine. at that though, do we need them sorted?

Reply:
They don't _need_ to be sorted.  I just adding sorting to mimic the web app export file.